Duplicati/Library/Main/Options.cs

```diff
@@ -1734,6 +1735,18 @@ public bool NoBackendverification
             get { return Library.Utility.Utility.ParseBoolOption(m_options, "no-backend-verification"); }
         }
                 
+        /// <summary>
+        /// Gets the percentage of samples to test during a backup operation
+        /// </summary>
+        public long BackupTestPercentage
+        {
+            get
+            {
+                m_options.TryGetValue("backup-test-percentage", out string s);
+                return string.IsNullOrEmpty(s) ? 0 : Math.Min(Math.Max(long.Parse(s), 0), 100);

```

Comment:
Can you validate the percentage? If parse throws exception or the number is negative, tell user has to be a number between 0 and 100. I know, negative gets mapped to 0, which should be ok.

Reply:
Good idea.  Revision 52790cc7a9759e5824466e2f870201e3175bbc54 validates the value and throws an exception if it is out of range.

Reply:
Ok, but Long.parse will throw a parsing exception if it's not a long. Can you catch it and output the message you added already?

Reply:
No problem.  Revision b4232ce1bfc1084eaa58704fb90ec21511044a47 catches any parse exception and provides a more meaningful message to the user.